StripePaymentSheet/StripePaymentSheet/Source/Helpers/PaymentSheetLinkAccount.swift

```diff
@@ -553,3 +553,20 @@ private extension LinkSettings.FundingSource {
         }
     }
 }
+
+// MARK: UpdatePaymentDetailsParams
+
+struct UpdatePaymentDetailsParams {
```

Comment:
this seems to be unused, is that expected?

Reply:
This was moved from a deleted file, as it's still used by the API. I *think* it might be used for inline Link enrollment, but we're not sure yet. Planning to do another code deletion pass for unused API bits at the end.